Zyantific.Zydis/Native/FormatterBuffer.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using Zyantific.Zycore.Native;
+using ZyanStatus = System.UInt32;
+using ZyanU64 = System.UInt64;
+using ZyanU8 = System.Byte;
+using ZyanUPointer = System.UIntPtr;
+using ZyanUSize = System.UIntPtr;
+using ZydisTokenType = System.Byte;
+namespace Zyantific.Zydis.Native
+{
+    using FormatterTokenConst = FormatterToken;
+
+    [StructLayout(LayoutKind.Sequential)]
+    public struct FormatterToken
+    {
+        ZydisTokenType type;
+
+        ZyanU8 next;
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+    public struct FormatterBuffer
+    {
+        [MarshalAs(UnmanagedType.I1)]
+        public bool IsTokenList;
+
+        public ZyanUSize Capacity;
+
+        public ZyanString String;
+
+        // the value parameter is supposed to be type char**, so I'm not sure if LPStr + ref is a valid substitution.
```

Comment:
Should be fine. Maybe `String` instead of `StringBuilder` would fit better tho.

Note to myself @flobernd: 
Change the ouput parameter in the C-source to (`const char*`)*. The string-data of the token shouldn't be mutable.